src/applications/personalization/profile-2/components/MilitaryInformation.jsx

```diff
@@ -175,10 +214,12 @@ MilitaryInformation.propTypes = {
       ),
     }).isRequired,
   }).isRequired,
+  veteranStatus: PropTypes.string,
 };
 
 const mapStateToProps = state => ({
   militaryInformation: state.vaProfile?.militaryInformation,
+  veteranStatus: state.user?.profile?.veteranStatus,
```

Comment:
would suggest using this using `selectProfile` selector https://github.com/department-of-veterans-affairs/vets-website/blob/bc8913736fe039ae5d092e568ed1e41c28ca8bba/src/platform/user/selectors.js#L8